docs/components/bucket.md

```diff
@@ -389,3 +392,71 @@ Flags:
       --single-run               Run replication only one time, then exit.
 
 ```
+
+### Downsample
+
+The downsample component of Thanos implements the downsample API on top of historical data in an object storage bucket. It continuously downsamples blocks in an object store bucket as a service.
+
+```bash
+$ thanos downsample \
```

Comment:
should this be `thanos bucket downsample`, matching usage below?

Reply:
@belm0, feel free to submit a PR which fixes this :P